.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
nit: can we pull in a standard `.gitignore` for nodejs? https://github.com/github/gitignore/blob/master/Node.gitignore

Reply:
done in https://github.com/urbanairship/node-connect-client/pull/7/commits/23cb6b50c1154bfd2013a0e57d15231ef731a765